spec/views/events/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
```

Comment:
view tests are hard to maintain and there is a better way to check the UI, you can remove the entire folder